tests/containers/containerd_rmt.pm

```diff
@@ -25,30 +23,37 @@ sub run {
 
     select_serial_terminal;
     systemctl 'stop firewalld';
+    ensure_ca_certificates_suse_installed();
     install_k3s();
     install_kubectl();
     install_helm();
 
-    my $rmt_helm = get_var('RMTTEST_REPO', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
+    my $helm_chart = get_var('HELM_CHART', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
     # pull in the testsuite
-    assert_script_run("curl -sSLk $rmt_helm | tar -zxf -");
-    my $rmt_config = get_var('RMTTEST_CONFIG', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
-    assert_script_run("curl -sSLOk $rmt_config");
-    assert_script_run("helm install rmt ./helm-charts-main/rmt-helm -f myvalue.yaml");
+    assert_script_run("curl -sSL $helm_chart | tar -zxf -");
```

Comment:
```suggestion
    assert_script_run("curl -sSL --retry 3 --retry-delay 30 $helm_chart | tar -zxf -");
```
Suggestion for improving the test stability. Alternatively we can also use `script_retry`.

Same for the `curl` call below